crates/eips/Cargo.toml

```diff
@@ -41,6 +41,9 @@ arbitrary = { workspace = true, features = ["derive"], optional = true }
 proptest = { workspace = true, optional = true }
 proptest-derive = { workspace = true, optional = true }
 
+# for signed authorization list arbitrary
+k256 = { workspace = true, optional = true }
```

Comment:
this is fine because if if the feature is enabled on alloy-primitives this dep is already part of the dep tree